.template/variants/web/spec/support/capybara.rb

```diff
@@ -27,7 +27,7 @@
   options.add_argument('no-sandbox')
 
   # Run headless by default unless CHROME_HEADLESS specified
-  options.add_argument('headless') unless ENV['CHROME_HEADLESS'] =~ /^(false|no|0)$/i
+  options.add_argument('headless') unless /^(false|no|0)$/i.match?(ENV['CHROME_HEADLESS'])
```

Comment:
Could you please explain more why you have to make this change?

I tried to add `rubocop-performance` to `Gemfile.tt` and require it in `.rubocop.yml` and create new rails app using this template.
After I run `rubocop --requrie rubocop-performance` there are only `FrozenStringLiteralComment` appear.

Thanks

Reply:
`rubocop-performance` warns this line @gutakk

Reply:
![image](https://user-images.githubusercontent.com/11751745/95421632-fa446580-0967-11eb-874a-024264d1deb8.png)

Reply:
thanks for explanation.